examples/http.rs

```diff
@@ -24,7 +23,7 @@ fn main() {
 
   let size = {
     let sr = fn_request(&request);
-    let (size, _buf) = length(sr)(&mut buffer).unwrap();
+    let (_buf, size) = sr(&mut buffer).unwrap();
```

Comment:
Is there a specific reason why you swapped the order in the tuple?

Reply:
Calling length directly put an emphasis on the length, so the length was the first elem of the tuple.
Now we provide the length as a secondary information for all generators, so I put it in second position.
As a bonus, it better matches what we had in cookie-factory 0.2

Reply:
Makes sense, I also prefer it. I was just wondering if there was any practical reason behind it too :)